components/cartlist-card/index.js

```diff
@@ -36,19 +36,19 @@ export const CartCard = (props) => {
         <button onClick={() => handleAddProduct(product.name)}>+</button>
         <button onClick={() => handleRemoveProduct(product.name)}>-</button>
       </div>
-      <span className="card-item"> RDS {product.price} </span>
+      <span className="card-item" style={{fontWeight:"bold"}}> RDS {product.price} </span>
```

Comment:
Can we avoid inline styling?
We can simply put the same styles in the `card-item` class if possible

Reply:
@whyDontI yes we can use "card-item" class for styling but it will apply to all elements having class "card-item" which we do not want. But we can use "id" for that particular element. So it will be good if I use id for that element to style it?

Reply:
You can use a second class for this style.

But this would be against using a common CSS class.
What is the purpose of a common class if we are not keeping all the `card-items` the same?

This should not be allowed unless there is a very good reason for it.

Reply:
@whyDontI ok, i'll make changes as per your suggestion. Thanks for explaination.

Reply:
@whyDontI changes has been made using new class. Please review.